Documentation/openshiftdocs/modules/clustermonitoringconfiguration.adoc

```diff
@@ -1,32 +1,35 @@
-:_content-type: ASSEMBLY
-include::_attributes/common-attributes.adoc[]
-:context: configmap-reference-for-cluster-monitoring-operator
+// DO NOT EDIT THE CONTENT IN THIS FILE. It is automatically generated from the 
+	// source code for the Cluster Monitoring Operator. Any changes made to this 
+	// file will be overwritten when the content is re-generated. If you wish to 
+	// make edits, read the docgen utility instructions in the source code for the 
+	// CMO.
+	:_content-type: ASSEMBLY
 
 == ClusterMonitoringConfiguration
 
 === Description
 
-ClusterMonitoringConfiguration defines configuration that allows users to customise the platform monitoring stack through the cluster-monitoring-config ConfigMap in the openshift-monitoring namespace
+The `ClusterMonitoringConfiguration` resource defines settings that customize the default platform monitoring stack through the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
 
 
 [options="header"]
 |===
 | Property | Type | Description 
-|alertmanagerMain|*link:alertmanagermainconfig.adoc[AlertmanagerMainConfig]|AlertmanagerMainConfig defines configuration related with the main Alertmanager instance.
+|alertmanagerMain|*link:alertmanagermainconfig.adoc[AlertmanagerMainConfig]|The `AlertmanagerMainConfig` resource defines settings for the Alertmanager component in the `openshift-monitoring` namespace.
```

Comment:
It's better change to below which is consistent with the descriptions of other components
`AlertmanagerMainConfig` defines settings for the Alertmanager component in the `openshift-monitoring` namespace.

Reply:
changed

Reply:
fixed